packages/lab/src/pagination/Pagination.tsx

```diff
@@ -58,30 +82,27 @@ export const Pagination = forwardRef<HTMLElement, PaginationProps>(
       () => ({
         page: pageState,
         count,
-        compact,
         onPageChange,
         paginatorElement,
         setPaginatorElement,
       }),
-      [pageState, count, onPageChange]
+      [pageState, count, onPageChange, paginatorElement]
     );
 
-    const onKeyDown: KeyboardEventHandler = useCallback(
-      ({ altKey, key }) => {
-        if (altKey) {
-          switch (key) {
-            case "PageDown":
-              onPageChange(Math.min(pageState + 1, count));
-              break;
-            case "PageUp":
-              onPageChange(Math.max(pageState - 1, 1));
-              break;
-            default:
-          }
+    const onKeyDown: KeyboardEventHandler = (event) => {
+      const { altKey, key } = event;
```

Comment:
There's not really much benefit in destructuring here over just using event.altKey and event.key